packages/codemodel.go/src/type.ts

```diff
@@ -138,6 +140,7 @@ export interface BytesType {
 // PrimitiveType is a Go integral type
 export interface PrimitiveType {
   typeName: PrimitiveTypeName;
+  encodeAsString?: boolean;
```

Comment:
Can you remove the `?` as I believe that `false` and `undefined` have the same meaning.